internal/store/bigtable/group.go

```diff
@@ -32,9 +32,9 @@ var groupRank = map[string]int{
 	"ipcc":       2,
 	"biomedical": 3,
 	"disaster":   4,
-	"place":   		100,
 	"borgcron":   10000, // Used for legacy base cache
 	"infrequent": 10000,
+	"place":      11000,
```

Comment:
Is this the final step for the migration?

Reply:
Correct.

Reply:
I guess the final final step is to roll out to prod... @pradh